kit/inputs/src/UIMaterialTextView/FloatingLabel.tsx

```diff
@@ -196,19 +202,23 @@ export const FloatingLabel: React.FC<FloatingLabelProps> = (props: FloatingLabel
             ],
         };
     });
+    const labelContainerOpacityStyle = useAnimatedStyle(() => {
+        return {
+            opacity: labelOpacity.value,
+        };
+    });
 
     if (!validateChildren(children)) {
         return null;
     }
     return (
         <View style={styles.container} pointerEvents="none">
-            <Animated.View
-                style={[styles.floatingLabel, labelContainerStyle]}
-                onLayout={onLabelLayout}
-            >
-                <Label animatedPosition={animatedPosition} labelOpacity={labelOpacity}>
-                    {children}
-                </Label>
+            <Animated.View style={labelContainerOpacityStyle}>
+                <Animated.View style={labelContainerStyle}>
```

Comment:
Do you remember why we made it separate? Bad we didn't leave a comment...

Reply:
Yes, opacity style didn't work on web with transform style in one `Animated.View`. But I checked it now and this problem doesn't actual anymore. I will remove this separation in next PR.